src/asciidoc/docs/develop/editor.adoc

```diff
@@ -1,123 +1,156 @@
-= The Koji editor
+= The Koji Code Editor
 :page-slug: editor
-:page-description: Guide for the Koji editor, a browser-based IDE for developing and testing Koji templates.
+:page-description: Guide for the Koji Code Editor, a browser-based IDE for developing and testing a first-generation Koji.
 :includespath: ../_includes
 
+The Koji Code Editor is a browser-based integrated development environment (IDE) that you can use to develop and test a first-generation Koji.
+The Code Editor provides easy access to the following:
+[.singlespaced]
```

Comment:
I'll chat with you before removing this. It looks really odd when very short bullets are so far apart from each other (which is the default style).

Reply:
With singlespaced style:
![PR161-with-singlespaced-style](https://user-images.githubusercontent.com/82409235/117521709-0f1bc280-af64-11eb-99a8-65ca3997ba08.png)

Without singlespaced style:
![PR161-without-singlespaced-style](https://user-images.githubusercontent.com/82409235/117521734-25c21980-af64-11eb-88ad-fa652e2c3a16.png)

Reply:
These bullets are too close together. If we want to adjust the bullet spacing, let's do it everywhere. Otherwise, let's not have multiple styles for the same element that are randomly applied. Please remove this style and all instances of it.

Reply:
Removed.